boot/bootdata/hivesft.inf

```diff
@@ -467,6 +469,13 @@ HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\HideDesktopIcons\Classi
 HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\BitBucket",,0x00000012
 HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\BitBucket\Volume",,0x00000012
 
+; For cleanmgr.exe
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\VolumeCaches",,0x00000012
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\VolumeCaches\Old ChkDsk Files",,0x00000012
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\VolumeCaches\Temporary Files",,0x00000012
+HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\VolumeCaches\RAPPS Files",,0x00000012
```

Comment:
?

Reply:
@learn-more This is where Disk Cleanup Handlers are stored. Currently, only cleanup profiles (or StageFlags) have been implemented.
https://docs.microsoft.com/pl-pl/windows/win32/lwef/disk-cleanup